src/configuration/sales/sales.md

```diff
@@ -36,6 +36,12 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Sale
 |--- |--- |--- |
 |Allow Reorder|Store View|Determines if the customers can reorder from their accounts. Options: Yes / No|
 
+## Allow Zero GrandTotal
+
+|Field|[Scope]({% link configuration/scope.md %})|Description|
```

Comment:
"Grand Total" and "Credit memo" are both two words.

Reply:
Hi, @dobooth 
Done! thank you😊